frontend/views/containers/chatroom/ChatMixin.js

```diff
@@ -141,6 +142,22 @@ const ChatMixin: Object = {
       if (to) {
         this.redirectChat(to)
       }
+    },
+    'ourGroupDirectMessages': {
+      immediate: true,
+      handler (to, from) {
+        // NOTE: whenever any group members leave the group, and the ourGroupDirectMessage is changed
+        //       currentChatRoomId needs to be considered to needs to be changed
+        //       if the currentChatRoomId (if it's DM) is no longer group direct message
+        if (Object.keys(this.ourDirectMessages).includes(this.currentChatRoomId)) {
```

Comment:
Should this be `Object.keys(from)`?

Reply:
No. It is `this.ourDirectMessages`, not `this.ourGroupDirectMessages`. Since there could be confusion, I will update some related codes.